CHANGELOG.md

```diff
@@ -5,6 +5,98 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+On the documentation side the API documentation has also been consolidated and there is now a unified and auto-generated [API documentation](https://github.com/ethereumjs/ethereumjs-vm/blob/master/docs/index.md) (previously being manually edited (and too often forgotten) in ``README``).
+
+- Added API tests for ``index.js``, ``StateManager``, see PR [#364](https://github.com/ethereumjs/ethereumjs-vm/pull/364)
+- Added API Tests for ``runJit`` and ``fakeBlockchain``, see PR [#331](https://github.com/ethereumjs/ethereumjs-vm/pull/331)
+- Added API tests for ``runBlockchain``, see PR [#336](https://github.com/ethereumjs/ethereumjs-vm/pull/336)
+- Added ``runBlock`` API tests, see PR [#360](https://github.com/ethereumjs/ethereumjs-vm/pull/360)
+- Added ``runTx`` API tests, see PR [#352](https://github.com/ethereumjs/ethereumjs-vm/pull/352)
+- Added API Tests for the ``Bloom`` module, see PR [#330](https://github.com/ethereumjs/ethereumjs-vm/pull/330)
+- New consistent auto-generated [API documentation](https://github.com/ethereumjs/ethereumjs-vm/blob/master/docs/index.md), see PR [#377](https://github.com/ethereumjs/ethereumjs-vm/pull/377)
+- Blockchain tests now run by default on CI, see PR [#374](https://github.com/ethereumjs/ethereumjs-vm/pull/374)
+- Switched from ``istanbul`` to ``nyc``, see PR [#334](https://github.com/ethereumjs/ethereumjs-vm/pull/334)
+- Usage of ``sealEngine`` in blockchain tests, see PR [#373](https://github.com/ethereumjs/ethereumjs-vm/pull/373)
+- New ``tap-spec`` option to get a formatted test run result summary, see [README](https://github.com/ethereumjs/ethereumjs-vm#running-tests-with-a-reporterformatter), see PR [#363](https://github.com/ethereumjs/ethereumjs-vm/pull/363)
+- Updates/fixes on the JSDoc comments, see PRs [#362](https://github.com/ethereumjs/ethereumjs-vm/pull/362), [#361](https://github.com/ethereumjs/ethereumjs-vm/pull/361)
+
+### Bug Fixes and Maintenance
+
+Some bug fix and maintenance updates:
+
+- Fix error handling in ``fakeBlockChain``, see PR [#320](https://github.com/ethereumjs/ethereumjs-vm/pull/320)
+- Update of ``ethereumjs-util`` to [v6.0.0](https://github.com/ethereumjs/ethereumjs-util/releases/tag/v6.0.0), see PR [#369](https://github.com/ethereumjs/ethereumjs-vm/pull/369)
+
+
+### Special Thanks
+
+Special thanks to:
+
+- @mattdean-digicatapult for his indefatigable work on the new StateManager interface and for fixing a large portion of the failing blockchain tests
+- @rmeissner for the work on Constantinople 
+- @vpulim for jumping in so quickly and doing a reliable ``SSTORE`` implementation within 4 days
+- @s1na for the new API test suite
+
+Many others did valuable contributions which can't all be listed here, many thanks to all!
```

Comment:
Curious: why it cannot be listed?

One possible way is what Solidity is doing, just listing all the user handles.

Reply:
You're right, will update.